app/filters.py

```diff
@@ -25,6 +25,17 @@ def datetime_format(value: str) -> str:
     return formatted_date
 
 
+def current_datetime_after_given(value: str) -> bool:
+    today = datetime.today().now()
+    parsed = datetime.strptime(value, "%Y-%m-%d %H:%M:%S")
+    return today > parsed
```

Comment:
Don't think we need to parse the string like that, can we try 
`datetime.fromisoformat(value)` instead?